audits/supersimple.py

```diff
@@ -0,0 +1,175 @@
+import math
+from scipy import stats
+from audits.audit import RiskLimitingAudit
+
+class SuperSimple(RiskLimitingAudit):
+
+    def __init__(self, risk_limit):
+        self.l = 0.5
```

Comment:
I recommend against using the "lambda" approach (in which the tolerable rate of 1-vote overstatements equals lambda times the diluted margin); if you do use it, I'd use a much smaller value of lambda, but it's probably better to estimate "expected" 1-vote overstatements somewhat akin to https://www.stat.berkeley.edu/~stark/Vote/auditTools.htm.